### ISSUE TITLE
CLI authToken receives 401 when trying to publish, used to work

### ISSUE THREAD

**Issue by kurbar** [kuoruan/verdaccio-openid#5]
### Versions
verdaccio v6.0
verdaccio-openid v0.10.1

Due to a secret rotation my Verdaccio instance was restarted over a year of running. With it I also updated Verdaccio to v6.0 and the plugin got updated to v0.10.1.

However something has changed internally since I'm unable to publish anything anymore. Login via web and npx command is successful (no errors in verdaccio logs) but when I try to publish a package, I get a 401 error and in verdaccio the logs say:
```
2024-10-07T10:27:07.240321947Z info <-- 10.244.1.70 requested 'PUT /@redacted%2fredacted'
2024-10-07T10:27:07.240394654Z warn --- "undefined" is not allowed to unpublish "@redacted/redacted"
2024-10-07T10:27:07.240400722Z error--- undefined is forbidden publish for @redacted/redacted
2024-10-07T10:27:07.240478787Z http <-- 401, user: null(10.110.16.3 via 10.244.1.70), req: 'PUT /@redacted%2fredacted', error: authorization required to publish package @redacted/redacted
2024-10-07T10:27:07.242440458Z http <-- 401, user: null(10.110.16.3 via 10.244.1.70), req: 'PUT /@redacted%2fredacted', error: authorization required to publish package @redacted/redacted
```

I have permissions configured so that `$authenticated` are allowed to do actions.
I'm using Azure OIDC authentication.

**Comment by kuoruan** [comment]
Can you paste your token to https://jwt.io/, check if the payload contains groups.

**Comment by kurbar** [comment]
Hmm, I might have a misconfiguration somewhere. `npx verdaccio-openid@0.6.2` does not save a JWT in `.npmrc`.

This probably is related to the legacy token signature? I currently have Verdaccio downgraded to v5.26.2 (which I had previously) and there publishing works fine.

**Comment by kuoruan** [comment]
OK, I'm checking it with legacy token signature.

**Comment by kuoruan** [comment]
Fixed, try v0.10.2
